packages/experimental/NativeDatePicker/ios/DatePickerManager.swift

```diff
@@ -27,13 +27,9 @@ public class DatePickerManager: NSObject {
         callback: @escaping RCTResponseSenderBlock
     ) {
         DispatchQueue.main.async {
-            // We need a UIViewController to present the actual DateTimePicker
-            // view.  With the advent of multi-window in iOS 13, we shouldn't
-            // use UIApplication.shared.keyWindow?.rootViewController, because
-            // keyWindow is deprecated.  We need to pick one of what could be
-            // multiple key windows.
-            guard let viewController = UIApplication.shared.windows.filter({$0.isKeyWindow}).first?.rootViewController else {
-                fatalError("Unable to get a UIViewController from current shared application context.")
+            // We need a UIViewController to present the actual DateTimePicker view.
+			guard let viewController = RCTPresentedViewController() else {
+                fatalError("Unable to get the current UIViewController from React Native.")
```

Comment:
unrelated to the current change but somewhat important: we need to avoid fatalError. It encodes the path of the source file in the binary which is not ok from a security perspective. Should use preconditionFailure instead. A pretty good writeup - https://lapcatsoftware.com/articles/fatalError.html

Reply:
Wow.  That's not good.